### ISSUE TITLE
Game freezes whenever it has to play a wmv file

### ISSUE THREAD

**Issue by joaomiguel-sb** [pythonlover02/Proton-Sarek#15]
As the title says, when a game has to play a wmv file, it simply freezes, and I have to close it via Steam, I managed to work around it by changing the videos to dummy files, but I'm not sure if this is feasible for every game. MP4 files play just fine.
Proton-GE just skips the files it seems.
Tested it with GGXX Accent Core Plus R and Melty Blood Type Lumina

**Comment by pythonlover02** [comment]
It looks like System Arc Works games aren't that Sarek friendly, huh? Take #14, for example. Anyway, I’ll see if I can fix it.

Have you tried running the game with the original WMV files using WineD3D? If it runs with GE-Proton, then it's something strange, to say the least, as Sarek is only a patched version of GE-Proton. I don't touch anything media-related, just change some lines of code related to DXVK and add my own features like software rendering parameters.

**Comment by joaomiguel-sb** [comment]
Yeah, lol, I also had the same save issue with Melty Blood Type Lumina recently which is a French Bread game.
As you said, with WineD3D it simply skips the video, I'm guessing it also crashes  on regular Proton-GE using vulkan, but I can't test it, since I can only use WineD3D on it.

**Comment by pythonlover02** [comment]
![image](https://github.com/user-attachments/assets/f48cee4a-3081-4727-8a81-3812eb7c64e6)
It appears that the crash occurs on Proton/GE-Proton when using the DXVK 1.10.x branch, indicating an issue on the DXVK side. I'll see if I can backport a fix for this to [DXVK-Sarek](https://github.com/pythonlover02/DXVK-Sarek), as Proton-Sarek will rely on versions from that repository on the future, rather than the current official v1.10.3 release.

**Comment by pythonlover02** [comment]
Also, could you make a list of games that suffer from this issue? I could add a workaround to use WineD3D by default for those games.

**Comment by joaomiguel-sb** [comment]
Sure, I can test some games that I think have wmv files, I'll report back when it's done

**Comment by pythonlover02** [comment]
Hello? Someone there?

**Comment by joaomiguel-sb** [comment]
Sorry about that, this is going to sound dumb, but I've been on vacation for a while and my linux laptop is at home, so this will have to wait a bit.

**Comment by pythonlover02** [comment]
Hello? Someone there?

**Comment by joaomiguel-sb** [comment]
Yes, my steam library is quite big, so it's taking a while, I actually found one game that just works for some reason, so I'm trying out games that I remember having cutscenes that are video files and checking on protondb if they explicitly mention any sort of video file format.

**Comment by pythonlover02** [comment]
![image](https://github.com/user-attachments/assets/e3a23fff-7034-44db-96be-31b51671538b)
![image](https://github.com/user-attachments/assets/f9a236f8-83a4-4d6a-9f6c-c867d846b9b4)

It works on my end. Could you test if it works on your end now?
I think that by mistake i patch it on DXVK-Sarek lol

**Comment by joaomiguel-sb** [comment]
It crashed, unless I made it use WineD3D which plays the video file but not the audio, weirdly enough.

**Comment by pythonlover02** [comment]
> It crashed, unless I made it use WineD3D which plays the video file but not the audio in it, weirdly enough.

._. Well, I guess we will continue with the idea of using WineD3D by default for these games. Are there any other games that crash when running the wmv files?

**Comment by joaomiguel-sb** [comment]
Devil May Cry 3 Special Edition (Crashes)
Devil May Cry 4 Special Edition (Crashes)
Devil May Cry HD Collection (Crashes)
Onimusha Warlords (Crashes even with WineD3D)
One Piece Pirate Warriors 3 (Crashes)
Melty Blood Type Lumina (Crashes)
Guilty Gear Accent Core Plus R (Crashes)
Blazblue Centralfiction (Crashes)

There's also two games that just work for some reason

**Comment by pythonlover02** [comment]
@joaomiguel-sb those games will run by default under WineD3D thanks to this [commit](https://github.com/pythonlover02/Proton-Sarek/commit/8c7ea62bf58f6bf295f6df893291bf1af60d129a) to the experimental branch
It should be on one of the future releases. Thanks for the help *:)*
